libs/core/src/lib/feed-input/feed-input.component.ts

```diff
@@ -0,0 +1,51 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ContentChild,
+    Input,
+    ViewEncapsulation,
+    AfterContentInit,
+    forwardRef,
+    OnDestroy, ViewChild
+} from '@angular/core';
+import { Subscription } from 'rxjs';
+import { ButtonComponent, FeedInputButtonDirective, FeedInputTextareaDirective } from '../..';
```

Comment:
Could you add direct imports there? Otherwise Travis CI is going to fail
![image](https://user-images.githubusercontent.com/26483208/99381990-21c61080-28cc-11eb-8ca5-a6a093115cbc.png)